pennylane/operation.py

```diff
@@ -1163,7 +1163,7 @@ def __neg__(self):
     def __pow__(self, other):
         r"""The power operation of an Operator object."""
         if isinstance(other, numbers.Number):
-            return qml.ops.Pow(base=self, z=other)  # pylint: disable=no-member
+            return qml.pow(self, z=other)  # pylint: disable=no-member
```

Comment:
`test_prod.py` and `test_sprod.py` also use `qml.ops.Pow`, would be nice to change it to `qml.pow`.

Reply:
Also, we can now remove the `# pylint: disable=no-member` comment.